components/text-input-field/TextInputField.style.js

```diff
@@ -6,6 +6,15 @@ const TextLabel = styled(P)`
   color: ${colors.BROWN};
   display: block;
   margin: 0;
+  padding-left: ${(props) => (props.required ? 0 : `2em`)};
```

Comment:
Looked at the deployed contact page, wouldn't we want 0 for padding-left if there is no star?

Reply:
Thanks! Will work on fixing it